scripts/check_headers.py

```diff
@@ -0,0 +1,61 @@
+import os
+import re
+import sys
+
+
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 python3 check_headers.py
+if os.getenv('NO_COLOR') is None:
+    header = '\033[1;33m'
+    reset = '\033[0m'
+else:
+    header = ''
+    reset = ''
+
+
+def print_colorful(text):
+    global header
+    global reset
+    print('{}{}{}'.format(header, text, reset))
+
+
+# Returns true if the file passes validation, false if it fails
+def is_file_compliant(regex, path):
+    if not path.endswith('.go'):
+        return True
+
+    f = open(path, 'r')
+    combined = ''
+    for i in range(number_of_lines):
+        combined += f.readline()
```

Comment:
I'm only suggesting this because seeing string += makes me twitchy (although I realise it's just 4 lines), and also I really miss Python sometimes.
```suggestion
    combined = ''.join(itertools.islice(f, number_of_lines))
```

Reply:
Good suggestion